Microsoft.AVS.Management/Microsoft.AVS.Management.psm1

```diff
@@ -1655,3 +1655,60 @@ function Set-ToolsRepo
         }
     }
 }
+
+<#
+.Synopsis
+    Set vSAN compression and deduplication on a cluster or clusters. If deduplication is enabled then compression is required.
+    The default cluster configuration is deduplication and compression but the customer can change that.
+    Choosing neither compression nor deduplication will disable both.
+    This requires action on every physical disk and will take time to complete.
+.EXAMPLE
+    Set-vSANCompressDedupe -ClustersToChange "cluster-1,cluster-2" -Compression $true
+    Set-vSANCompressDedupe -ClustersToChange "cluster-1,cluster-2" -Deduplication $true
+    Set-vSANCompressDedupe -ClustersToChange "cluster-1,cluster-2"
+    Set-vSANCompressDedupe -ClustersToChange "*"
+#>
+function Set-vSANCompressDedupe
+{
+    [AVSAttribute(60, UpdatesSDDC = $true)]
+    param(
+        [Parameter(Mandatory = $true)]
+        [String]$ClustersToChange,
+        [Parameter(Mandatory = $false,
+        HelpMessage = "Enable compression and deduplication.")]
+        [bool]$Deduplication = $false,
+        [Parameter(Mandatory = $false,
+        HelpMessage = "Enable compression only.")]
+        [bool]$Compression = $false
+    )
+
+    # $cluster is an array of cluster names or "*""
+    foreach ($cluster_each in ($ClustersToChange.split(",",[System.StringSplitOptions]::RemoveEmptyEntries)).Trim())
+    {
+        $Clusters += Get-Cluster -Name $cluster_each
+    }
+
+    foreach ($Cluster in $Clusters)
+    {
+        $cluster_name = $Cluster.Name
+
+        If ($Deduplication -eq $true)
```

Comment:
You can probably drop `-eq $true` as `$Deduplication` is already a boolean expression.

Reply:
I removed the default values and the "-eq $true"